src/target/unix.rs

```diff
@@ -32,6 +33,12 @@ pub(crate) fn launch(path: &str) -> Result<Pid, Box<dyn std::error::Error>> {
         ForkResult::Child => {
             ptrace::traceme()?;
 
+            // Disable ASLR
+            #[cfg(target_os = "linux")]
+            unsafe {
+                libc::personality(0x0040000 /*ADDR_NO_RANDOMIZE*/);
```

Comment:
Nit: this probably would be better as
```suggestion
               const ADDR_NO_RANDOMIZE: usize = 0x0040000;
               libc::personality(ADDR_NO_RANDOMIZE);
```
It also seems that this const is missing in rust-lang/libc, I'm going to raise a PR for that :)

Reply:
Yeah, all personalities are missing. I had to `cat /usr/include/sys/personality.h` to find the correct value.